process/resultsParser/wellKnown.go

```diff
@@ -0,0 +1,23 @@
+package resultsParser
+
+type (
+	// WellKnownTopics is an enum that contains the most common topics encapsulated in a smart contract.
+	WellKnownTopics = string
+
+	// WellKnownEvents is an enum that contains the most common events encapsulated in a smart contract.
+	WellKnownEvents = string
+)
+
+const (
+	// OnTransactionCompleted is the event when a transaction is completed.
+	OnTransactionCompleted WellKnownEvents = "completedTxEvent"
```

Comment:
you can consts for these identifiers in `mx-chain-core-go` no need to redefine her
please refactor

Reply:
https://github.com/multiversx/mx-chain-core-go/blob/7d0b8a1b82d34c5689022cdcd27b85f0a4d4e1ff/core/constants.go#L253

Reply:
did refactor. but the `too much gas warning` is not part of that package. Should I define it there or keep it defined here?

Reply:
For the moment you can keep it defined here, also please remove unused consts.